adyenv6backoffice/resources/adyenv6backoffice-backoffice-config.xml

```diff
@@ -25,21 +25,28 @@
 					<editorArea:attribute qualifier="adyenMerchantAccount"/>
 					<editorArea:attribute qualifier="adyenImmediateCapture"/>
 					<editorArea:attribute qualifier="adyenAllowedCards"/>
-                    <editorArea:attribute qualifier="adyenRecurringContractMode"/>
+					<editorArea:attribute qualifier="adyenRecurringContractMode"/>
 					<editorArea:attribute qualifier="adyenTestMode"/>
-                </editorArea:section>
-                <editorArea:section name="hmc.adyen.api">
-                    <editorArea:attribute qualifier="adyenAPIEndpointPrefix"/>
-                    <editorArea:attribute qualifier="adyenAPIKey" editor="com.hybris.cockpitng.editor.defaultpassword"/>
-                </editorArea:section>
-				<editorArea:section name="hmc.adyen.hpp">
-					<editorArea:attribute qualifier="adyenSkinCode"/>
-					<editorArea:attribute qualifier="adyenSkinHMAC" editor="com.hybris.cockpitng.editor.defaultpassword"/>
+				</editorArea:section>
+				<editorArea:section name="hmc.adyen.api">
+					<editorArea:attribute qualifier="adyenAPIEndpointPrefix"/>
+					<editorArea:attribute qualifier="adyenAPIKey" editor="com.hybris.cockpitng.editor.defaultpassword"/>
 				</editorArea:section>
 				<editorArea:section name="hmc.adyen.notifications">
 					<editorArea:attribute qualifier="adyenNotificationUsername"/>
 					<editorArea:attribute qualifier="adyenNotificationPassword" editor="com.hybris.cockpitng.editor.defaultpassword"/>
 				</editorArea:section>
+				<editorArea:section name="hmc.adyen.pos">
+					<editorArea:attribute qualifier="adyenPosEnabled"/>
+					<editorArea:attribute qualifier="adyenPosApiKey" editor="com.hybris.cockpitng.editor.defaultpassword"/>
+					<editorArea:attribute qualifier="adyenPosMerchantAccount"/>
+					<editorArea:attribute qualifier="adyenPosStoreId"/>
+					<editorArea:attribute qualifier="adyenPosRecurringContractMode"/>
+				</editorArea:section>
+				<editorArea:section name="hmc.adyen.hpp">
+					<editorArea:attribute qualifier="adyenSkinCode"/>
+					<editorArea:attribute qualifier="adyenSkinHMAC" editor="com.hybris.cockpitng.editor.defaultpassword"/>
+				</editorArea:section>
```

Comment:
where do we need adyenSkinCode and adyenSkinHMAC for ?

Reply:
It is old code and clean up of this is not part of this story.

Reply:
no it is just formatting change.